README.md

```diff
@@ -69,6 +105,17 @@ Get only last 10 messages
 RootResult<MessageResponse> last10Messages = client.getMessage().list(10, 1);
 System.out.println(last10Messages);
 ```
+The next method returns the next item in a list.
+```java
+RootResult<MessageResponse> messageResponse = client.getMessage().next();
+System.out.println(last10SenderNameResponse);
+```
+The previous method returns the previous item in the list.
+```java
+RootResult<MessageResponse> messageResponse = client.getMessage().previous();
+System.out.println(last10SenderNameResponse);
```

Comment:
Same

Reply:
copy paste thing..
forgot to rename them..